README.md

```diff
@@ -15,83 +16,212 @@ npm run test
 
 This project works a bit differently than a regular test suite for now. We have Jest installed more as a test runner, but we don't really use assertions for now. We use it more to gather and report numbers.
 
+### FCP
+
+First Contentful Paint (FCP) "measures the total time taken from the beginning of a page load to the point any content is rendered on the screen" (see https://web.dev/fcp/).
+To get FCP we use the `PageLoadingMetrics` function from `TraceModel`.
+
+### DCL
+
+DOM Content Loaded (DCL) "marks the point when both the DOM is ready and there are no stylesheets that are blocking JavaScript execution" (see https://web.dev/critical-rendering-path-measure-crp/). In other words, it measures the time when the HTML document has been completely loaded and read.
+To get DCL we use the `PageLoadingMetrics` function from `TraceModel`.
+
+### LCP
+
+The Largest Contentful Paint (LCP) metric "reports the render time of the largest image or text block visible within the viewport, relative to when the page first started loading" (see https://web.dev/lcp/).
+To get LCP we use the `PageLoadingMetrics` function from `TraceModel`.
+
 ### FPS
 
+Frames per second (FPS) measures video quality (how many images are displayed consecutively each second). We use it to measure component animation performance.
 To gather average FPS time, we just use the `FPS` function from `TraceModel`.
-For FPS we don't need to specify any components to look for as it will just grab the SPF for the entire page.
+For FPS we don't need to specify any components to look for as it will just grab the FPS for the entire page.

```

Comment:
It doesn't seem to right description currently. 
Isn't the fps from requestAnimationFrame ?
CC: @daniel-stoian-lgp